sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -172,20 +174,42 @@ export class SupportingUserController {
           return job.error(SUPPORTING_USER_FULL_NAME_NOT_RESOLVED, message);
         }
       }
+      const isAbleToReport = job.variables.isAbleToReport;
       return this.dataSource.transaction(
-        // Transaction created with the expectation that a notification will also be created
-        // for the supporting user and it should be saved in the same transaction.
         async (entityManager: EntityManager) => {
           const createdSupportingUserId =
             await this.supportingUserService.createIdentifiableSupportingUser(
               {
                 applicationId: job.variables.applicationId,
                 supportingUserType: job.variables.supportingUserType,
                 fullName,
-                isAbleToReport: job.variables.isAbleToReport,
+                isAbleToReport,
               },
               entityManager,
             );
+          const notificationPayload = {
+            givenNames: application.student.user.firstName,
+            lastName: application.student.user.lastName,
+            toAddress: application.student.user.email,
+            userId: application.student.user.id,
+            supportingUserType:
+              job.variables.supportingUserType === SupportingUserType.Parent
+                ? "parent"
+                : "partner",
+            parentFullName: fullName,
+            applicationNumber: application.applicationNumber,
+          };
+          if (isAbleToReport) {
+            await this.notificationActionsService.saveParentInformationRequiredFromParentNotification(
+              notificationPayload as ParentInformationRequiredFromParentNotification,
```

Comment:
Please avoiding force casting. It will hide any type errors in future. Defining the payload type this way alternatively fixes the typing.

<img width="1093" height="647" alt="image" src="https://github.com/user-attachments/assets/c710f5bd-a2de-4ccb-ad83-b85629e6d315" />